server/delay_server/src/irodsDelayServer.cpp

```diff
@@ -635,8 +635,10 @@ int main(int argc, char** argv)
         try {
             return irods::get_advanced_setting<const int>(irods::CFG_DELAY_SERVER_SLEEP_TIME_IN_SECONDS);
         }
-        catch (const irods::exception& e) {
-            logger::delay_server::error(e.what());
+        catch (...) {
+            logger::delay_server::warn("Could not retrieve sleep time from configuration. Delay server will "
```

Comment:
consider showing the actual entry name that was missing - so the admin can just copy/paste to fix?

Reply:
done